src/sentry/utils/platform_categories.py

```diff
@@ -19,6 +19,7 @@
     "javascript-astro",
     "unity",
 ]
+FRONTEND_SET = set(FRONTEND)
```

Comment:
do we have to create another variable? why can't they just be sets to begin with?

Reply:
These were being used in getsentry, so I didn't want to change them if I didn't have to, but it looks like there isn't any usage that requires a list specifically, and ordering doesn't seem to matter so they should be okay to be swapped.